lib/base-compiler.js

```diff
@@ -267,7 +267,7 @@ export class BaseCompiler {
                 timeoutMs: timeoutMs,
                 ldPath: _.union(this.compiler.ldPath, executeParameters.ldPath).join(':'),
                 input: executeParameters.stdin,
-                additionalEnv: executeParameters.additionalEnv,
+                env: executeParameters.env,
```

Comment:
Tested locally, as this worried me at first. Using firejail and this change, compiling and executing:

```cpp
#include <cstdio>

int main(int , const char *[], const char *envp[]) { 
    while (*envp) {
        printf("%s\n", *envp++);
    }
}
```

yielded
```
LD_LIBRARY_PATH=/opt/compiler-explorer/gcc-snapshot/lib:/opt/compiler-explorer/gcc-snapshot/lib32:/opt/compiler-explorer/gcc-snapshot/lib64
ASAN_OPTIONS=color=always
UBSAN_OPTIONS=color=always
MSAN_OPTIONS=color=always
LSAN_OPTIONS=color=always
```